Apps/Laboratory/src/Services/ILaboratoryService.cs

```diff
@@ -20,39 +20,37 @@ namespace HealthGateway.Laboratory.Services
     using System.Threading.Tasks;
     using HealthGateway.Common.Data.ViewModels;
     using HealthGateway.Laboratory.Models;
+    using HealthGateway.Laboratory.Models.PHSA;
 
     /// <summary>
-    /// The laboratory data service.
+    /// The Covid19 data service.
```

Comment:
The change to the documentation here is not required; we will be using the same service for other types of laboratory tests.